ClientSecurity/Configuring_IPS_Notifications.md

```diff
@@ -0,0 +1,61 @@
+# Configuring Intrusion Prevention System (IPS) Notifications
+
+## Overview
+
+The Platform CenturyLink IPS utilizes an Agent installed on your Virtual Machine (VM) that will monitor that VM for suspicious activity. If suspicious activity is found, the Agent will log it and may block or stop the activity, and will report it based on the IPS policy. There is a default policy associated to each VM that is automatically tuned based on the host operating system and installed applications.
+
+The Blueprint allows a customer that has purchased the IPS service from Platform CenturyLink to modify how they would like to be notified regarding IPS security events. This Blueprint will only change notification settings for the server it is run against.
+
+## Prerequisites
+•	A CenturyLink Cloud Account
+•	Managed or Unmanaged Operating System Services on the Virtual Machine, with the IPS Agent installed
+•	Slack channel & WebHook URL (See Utilizing SLACK for IPS Event Notifications)
+
+## Supported Managed Operating Systems
+•	Red Hat Enterprise Linux 5 (64-bit only)
+•	Red Hat Enterprise Linux 6 (64-bit only)
+•	Microsoft Windows Server 2008 (64-bit only)
+•	Microsoft Windows Server 2012 (64-bit only)
+
+## Installation Process
+
+1. Search for **Configure Notifications** in the Blueprint library. Then, click on the desired Operating System blueprint to configure Notifications.
+
+![Control Portal.png](https://ucarecdn.com/ed5616b2-5aec-4a96-8c81-b6084adf627a/)
+![CLC IPS - Configure.png](https://ucarecdn.com/87d87b6e-b3fc-461d-a271-3c678b231811/)
```

Comment:
should not be linking to external repository for images. your description on the text reader should not be the file name either. it should be a handy text that describes what the image is showing for a text reader. see the readme in github